RayLin.md

```diff
@@ -1,30 +1,29 @@
-# 09/05 ~ 09/09
+# 09/12 ~ 09/14
 
 - [Video Control]
   - Bug 1271765 - Visual refresh of media controls
-    - r?jaws
-    - a regression was found, so changed to r- from r+ on the weekend.
-      - fixed and ask review again.
-		- duration/position label localization issues fixed.
+    - r+jaws
+    - found a few test failures. (3 files)
+      - fixed 2 of 3
+      - We did a hack to determine whether plaform is mobile or desktop by deferring initialization process. Unfortunately, it conceptually opposes to SimpleTest's framework, and breaks one of the test cases. Need a workaround here.
```

Comment:
I think Jared could. 

Now the issue is fixed, and actually the code is reverted to my original approach to determine platform instead of Jared's. I think Jared would agree with this workaround if he noticed this problem. So I'm gonna update patch and ask Jared to review again. Supposing no refusal, this patch could land.

Thanks :)